frontend/src/pages/accounts/settings.page.tsx

```diff
@@ -173,10 +173,10 @@ const Settings: NextPage = () => {
               <p>{l10n.getString("setting-tracker-removal-note")}</p>
             </label>
           </div>
-          <aside role="alert" className={styles["field-warning"]}>
```

Comment:
This field became visible with the new mock data in the test, and then caught by the a11y checker - apparently `<aside>`s should be able to stand by themselves, independent of the rest of the content, which is not true for these warnings. Hence my replacing them with plain `<div>`s.